From 60f1d000300603afcd18ed4a27a4a4af374d867a Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Wed, 27 Feb 2019 13:11:22 +0100
Subject: [PATCH] BPF: UMH: do not detect if compiler can link userspace
 program.

References: boo#1127188
Patch-mainline: no, SUSE specific

SUSE crosscompilers cannot build userspace programs. They do not include
libc and related headers.

The configuration files are updated using such crosscompilers leading to
this UMH symbol flipping and build kernel build failures due to
configuration differences.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 net/bpfilter/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/bpfilter/Kconfig b/net/bpfilter/Kconfig
index e558b46596c4..2e9b6a8692ea 100644
--- a/net/bpfilter/Kconfig
+++ b/net/bpfilter/Kconfig
@@ -8,7 +8,6 @@ menuconfig BPFILTER
 if BPFILTER
 config BPFILTER_UMH
 	tristate "bpfilter kernel module with user mode helper"
-	depends on $(success,$(srctree)/scripts/cc-can-link.sh $(CC))
 	default m
 	help
 	  This builds bpfilter kernel module with embedded user mode helper
-- 
2.20.1

